config/default.yml

```diff
@@ -155,9 +155,14 @@ RSpec/BeEql:
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/BeEql
 
 RSpec/BeNil:
-  Description: Check that `be_nil` is used instead of `be(nil)`.
+  Description: Ensures a consistent style is used when matching `nil`.
   Enabled: pending
+  EnforcedStyle: be_nil
```

Comment:
On `real-world-rspec`:
```
$ rg --no-ignore --hidden --stats 'be_nil'
10881 matches

$ rg --no-ignore --hidden --stats 'be[ \(]nil'
1290 matches
```
The latter includes text matches in strings and comments (e.g. "cannot be nil when locking the attribute").

So 👍 for enforcing `be_nil`.